src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -121,7 +121,10 @@
     "abnormal_mechanism": PREFIX + 236,  # release health
     "anr_foreground": PREFIX + 237,  # release health
     "anr_background": PREFIX + 238,  # release health
-    "drop": PREFIX + 239,  # transactions. "decision" and "keep" already exist
+    # Transactions
+    "drop": PREFIX + 239,
+    "decision": PREFIX + 240,
+    "keep": PREFIX + 241,
```

Comment:
I think we only need `decision` in here.
I don't think we use indexes for `drop` and `keep` since we look at the raw_tags.
Am I missing something?